lib/grammers-crypto/src/aes.rs

```diff
@@ -12,7 +12,6 @@ use aes::cipher::{BlockDecrypt, BlockEncrypt, NewBlockCipher};
 pub fn ige_encrypt(plaintext: &[u8], key: &[u8; 32], iv: &[u8; 32]) -> Vec<u8> {
     assert!(plaintext.len() % 16 == 0);
     let mut ciphertext = vec![0; plaintext.len()];
-    assert!(ciphertext.len() % 16 == 0);
```

Comment:
This was here because I thought it would've made the compiler happier (but maybe not).

Reply:
Yea, apparently doing own asserts makes the compiler not do them later on in the loops, but here it is redundant.